Sources/PayPal/PayPalUIInterfaces/PayPalUIFlow.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+import UIKit
+@_implementationOnly import PayPalCheckout
+
+#if canImport(PaymentsCore)
+import PaymentsCore
+#endif
+
+protocol PayPalUIFlow {
```

Comment:
So these are only needed for testing purposes. For me, it's a little confusing managing both the name of the protocol, and the names of the implementation classes that it wraps / abstracts.

This is a pattern we use in `APIClientProtocol` and are sure to need / use often again in our SDK. Can we come up with a consistent naming pattern for these protocols - whose intent is just to abstract for testing?

I think `<NameOfThing>Protocol` or `<NameOfThing>Wrapper` something like that, would help with consistency and readability.

Reply:
that's a good callout. I'll update to be `CheckoutProtocol`

Reply:
I was updating some other things so updated this as well! 😄

Reply:
thank you jax!